debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -715,6 +803,70 @@ public Duration getLogMiningArchiveLogRetention() {
         return Duration.ofHours(getConfig().getLong(LOG_MINING_ARCHIVE_LOG_HOURS));
     }
 
+    /**
+     * 
+     * @return int The minimum SCN interval used when mining redo/archive logs
+     */
+    public int getLogMiningBatchSizeMin() {
+        return getConfig().getInteger(LOG_MINING_BATCH_SIZE_MIN);
+    }
+
+    /**
+     * 
+     * @return int Number of actual records that will be fetched from the log mining contents view
+     */
+    public int getLogMiningViewFetchSize() {
+        return getConfig().getInteger(LOG_MINING_VIEW_FETCH_SIZE);
+    }
+
+    /**
+     * 
+     * @return int The maximum SCN interval used when mining redo/archive logs
+     */
+    public int getLogMiningBatchSizeMax() {
+        return getConfig().getInteger(LOG_MINING_BATCH_SIZE_MAX);
+    }
+
+    /**
+     * 
+     * @return int The default SCN interval used when mining redo/archive logs
+     */
+    public int getLogMiningBatchSizeDefault() {
+        return getConfig().getInteger(LOG_MINING_BATCH_SIZE_DEFAULT);
+    }
+
+    /**
+     * 
+     * @return int The minimum sleep time used when mining redo/archive logs
+     */
+    public int getLogMiningSleepTimeMin() {
```

Comment:
Should return `java.time.Duration` (same below).

Reply:
Sure

Reply:
Like with the suffix comment above, I see the latest commit suggests the usage of `Duration` but the commit doesn't appear to include such changes.

Reply:
🤦‍♂️
Should have looked within the commit. Proper change was done on a different local fork yet of course the commit filenames matched. Anyways, pebkac, updated PR with the proper renaming and typing.